drivers/modem/Kconfig.cellular

```diff
@@ -36,4 +36,18 @@ config MODEM_CELLULAR_PERIODIC_SCRIPT_MS
 	int "Periodic script interval in milliseconds"
 	default 2000
 
+config MODEM_CELLULAR_UART_BUFFER_SIZES
+	int "The UART receive and transmit buffer sizes in bytes."
+	default 512
+
+config MODEM_CELLULAR_CMUX_MAX_FRAME_SIZE
+	int "The maximum CMUX frame size in bytes."
+	help
+	  This value affects the size of buffers used to receive and transmit CMUX frames.
```

Comment:
I recommend placing the help as a last item for the option (like in other kconfig options), now it is quite confusing to read as the default value is after the help.